samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/device_tests/DeviceScreenshotSampleTest.kt

```diff
@@ -32,48 +27,24 @@ class DeviceScreenshotSampleTest : TestCase() {
 
     @Test
     fun screenshotSampleTest() {
+        val screenshotsDir = device.screenshots.getScreenshotDir()
         before {
-            deleteDir(screenshotsDir())
+            deleteDir(screenshotsDir)
         }.after {
         }.run {
-
             step("Take a screenshot") {
-                assertFalse(screenshotExists())
-                device.screenshots.take(SCREENSHOT_TAG)
-                assertTrue(screenshotExists())
+                assertTrue(screenshotsDir.list()?.isEmpty() ?: true)
```

Comment:
Intention of the assertion has been changed

Reply:
How? We check that screenshot folder is empty, take screenshot and check that it exists as we did before